include/cocaine/rpc/slot/deferred.hpp

```diff
@@ -128,6 +128,12 @@ struct deferred {
         return *this;
     }
 
+    template<class UpstreamType>
+    void attach(UpstreamType&& upstream) const {
```

Comment:
Codestyle (missed it the first time, sorry): return types should be on a separate line, i.e.:

```
template<class UpstreamType>
void
attach(UpstreamType&& upstream) const { ...
```